GNUmakefile

```diff
@@ -25,23 +25,23 @@ tools: lint-tools test-tools generate-tools
 .PHONY: generate-tools
 generate-tools: ## Install the tools used to generate code
 	@echo "==> Installing code generate tools..."
-	GO111MODULE=on cd tools && go get github.com/bufbuild/buf/cmd/buf@v0.36.0
-	GO111MODULE=on cd tools && go get github.com/golang/protobuf/protoc-gen-go@v1.4.3
+	cd tools && go install github.com/bufbuild/buf/cmd/buf@v0.36.0
+	cd tools && go install github.com/golang/protobuf/protoc-gen-go@v1.4.3
 	@echo "==> Done"
 
 .PHONY: test-tools
 test-tools: ## Install the tools used to run tests
 	@echo "==> Installing test tools..."
-	GO111MODULE=on cd tools && go get gotest.tools/gotestsum@v0.6.0
+	cd tools && go install gotest.tools/gotestsum@v1.8.2
 	@echo "==> Done"
 
 .PHONY: lint-tools
 lint-tools: ## Install the tools used to lint
 	@echo "==> Installing lint tools..."
-	GO111MODULE=on cd tools && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
-	GO111MODULE=on cd tools && go get -u honnef.co/go/tools/cmd/staticcheck@2020.1.6
-	GO111MODULE=on cd tools && go get github.com/hashicorp/go-hclog/hclogvet@v0.1.3
-	GO111MODULE=on cd tools && go get github.com/hashicorp/hcl/v2/cmd/hclfmt@d0c4fa8b0bbc2e4eeccd1ed2a32c2089ed8c5cf1
+	cd tools && go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
+	cd tools && go install honnef.co/go/tools/cmd/staticcheck@2022.1.3
+	cd tools && go install github.com/hashicorp/go-hclog/hclogvet@v0.1.5
```

Comment:
The version bumps here were necessary to support the new Go version. I was getting weird errors until they were all upgraded.